src/explorer/ExplorerOptions.cs

```diff
@@ -2,7 +2,7 @@ namespace Explorer
 {
     public class ExplorerOptions
     {
-        public const int DefaultSamplesToPublish = 20;
+        public const int DefaultDefaultSamplesToPublish = 20;
```

Comment:
:smile:

Reply:
Actually this isn't a typo, just some awkward naming 🙈 

I renamed `SamplesToPublish` to `DefaultSamplesToPublish` (to distinguish it from the request param). So `DefaultSamplesToPublish` naturally becomes `DefaultDefaultSamplesToPublish`...